src/MicrosoftTeams.schema.json

```diff
@@ -182,7 +185,8 @@
                     "websiteUrl": {
                         "type": "string",
                         "description": "The url to point at if a user opts to view in a browser.",
-                        "maxLength": 2048
+                        "maxLength": 2048,
+                        "pattern": "^[Hh][Tt][Tt][Pp][Ss]://"
```

Comment:
Use "pattern": "^[Hh][Tt][Tt][Pp][Ss]?://" since this does't have to be an https URL